ext/sysvsem/sysvsem.c

```diff
@@ -266,7 +266,7 @@ PHP_FUNCTION(sem_get)
 /* }}} */
 
 /* {{{ php_sysvsem_semop */
-static void php_sysvsem_semop(INTERNAL_FUNCTION_PARAMETERS, int acquire)
+static void php_sysvsem_semop(INTERNAL_FUNCTION_PARAMETERS, const bool acquire)
```

Comment:
note: not sure the const qualifier for acquire argument is really useful, it s just a bool passed by value.